hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerController.java

```diff
@@ -140,4 +141,8 @@ private Handler getHandler(final Container container) {
   public Iterator<Container> getContainers() {
     return containerSet.getContainerIterator();
   }
+
+  public Iterator<Container> getContainersForVolume(String volumeUuid) {
```

Comment:
Instead of `String` can we use `HddsVolume` as method argument?
Then we can also rename the method to `public Iterator<Container> getContainers(HddsVolume volumeId)`

Reply:
Good idea. Done.